packages/api/test/e2e/api.v36-compat.e2e.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019-2020 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Api as ApiPromise } from "@cennznet/api";
+import { AssetInfo, AssetOptions, Hash } from "@cennznet/types";
+import { SubmittableResult } from '@polkadot/api';
+import { Keyring } from '@polkadot/keyring';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import config from '../../../../config';
+
+const keyring = new Keyring({ type: 'sr25519' });
+
+describe('runtime v36 compatibility', () => {
+  let api;
+  let alice, bob;
+  let stakingAssetId;
+
+  beforeAll(async () => {
+    await cryptoWaitReady();
+    const provider = process.env.TEST_TYPE?.toLowerCase() === 'integration' ? config.wsProvider.integrationV36 : 'ws://localhost:9944';
```

Comment:
Would be good to leave this one as 9945 to avoid false positives e.g if a 1.2.0 node is running here. I think this is what @alexsednz means

Reply:
ok